lib/components/dashboard/course-list/horizontal/Semester.tsx

```diff
@@ -197,6 +197,7 @@ const Semester: FC<{
     if (version !== 'None') {
       const body = {
         ...version,
+        _id: undefined,
```

Comment:
This should be fixed on dev. Please delete this to prevent a merge conflict on dev.